### DISCUSSION TITLE
OpenDTU und ModBus TCP

### DISCUSSION THREAD

**Discussion by pneutek** [tbnobody/OpenDTU#1511] [Q&A]
Hallo @ all

Ich bin neu hier, und plane eine PV mit Mikrowechselrichter. Kann open DTU die Daten per Modbus TCP für eine andere Steuerung bereitstellen? Ich benutze eine Steuerung für die WP von der Technischen Alternative. Das Netzwermodul (CMI) kann Modbus TCP.
Hier kann ich auch Daten loggen.

**Comment by astrakaro** [comment]
Hallo, ich suche auch nach der Funktion Daten via Modbus TCP vom Hoymiles WR zu pollen.
Habe die Firmware v24.1.26, jedoch erhalte ich keinen Response auf dem Port 502.
